packages/integration-tests/src/helpers.ts

```diff
@@ -31,10 +31,6 @@ export const registerNewUser = async (username: string, password: string) => {
 
   assert(client.interactionCookie, new Error('Session not found'));
 
-  if (!client.interactionCookie) {
-    return;
-  }
-
```

Comment:
Try to make sure: delete these lines due to duplication of logic in assert(client.interactionCookie, ...) ?

Reply:
yes might be useless